cql3/expr/expression.hh

```diff
@@ -617,6 +617,7 @@ extern expression search_and_replace(const expression& e,
         const noncopyable_function<std::optional<expression> (const expression& candidate)>& replace_candidate);
```

Comment:
GitHub still fails to allow comments on commit messages, so I'll place one here: the commit title contains a redundant `allow`, I think it should be `cql3: expr: allow prepare of tuple_constructor with no receiver`.